spec/spec_helper.rb

```diff
@@ -1,2 +1,12 @@
-$LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
-require "mxmnd"
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'mxmnd'
+require 'webmock/rspec'
+
+def presence_of_keys?(dict, keys_array)
+  keys_array.each { |key| return false unless dict[key] }
+  true
+end
+
+def hash_including_locales(dict, num_of_locales)
```

Comment:
this method doesn't care if second argument is locales or something else, a better way is to just bring this into the spec example, e.g. `expect(dict['name'].size).to eq 8`.
Any custom matching methods make spec reading more confusing.